Automattic/wp-calypso
# Unpin some dependencies

#### Changes proposed in this Pull Request

* Unpin some dependencies in package.json

#### Testing instructions

* Verify there are no packages removed/added in `yarn.lock`
* Validate the dependency tree has not changed:
    1. Install dependencies and generate the tree for this branch: `find . -type d -name node_modules | xargs rm -fr && yarn && NODE_OPTIONS=--max_old_space_size=8192 yarn run effective-module-tree -o list > ../branch-tree`
    2. Install dependencies and generate the tree for master (tested with c9aab3df58): `find . -type d -name node_modules | xargs rm -fr && yarn && NODE_OPTIONS=--max_old_space_size=8192 yarn run effective-module-tree -o list > ../master-tree`
    3. Diff both trees and check there are no differences: `diff ../branch-tree ../master-tree`
